### ISSUE TITLE
Broken version specific 'grpc' dependency in package.json for 'grpc-bchrpc-node'

### ISSUE THREAD

**Issue by upfs** [simpleledger/SLPDB#52]
Running `npm install`  yields error
```
../deps/grpc/src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc:30:10: fatal error: address_sorting/address_sorting.h: No such file or directory
 #include <address_sorting/address_sorting.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

due to `grpc-bchrpc-node` being version locked to `grpc@1.23.1 ` for which pre-built binaries are not available 

```
node-pre-gyp WARN Tried to download(404): https://node-precompiled-binaries.grpc.io/grpc/v1.23.1/node-v79-linux-x64-glibc.tar.gz                                                                                                                            
node-pre-gyp WARN Pre-built binaries not found for grpc@1.23.1 and node@13.0.1 (node-v79 ABI, glibc) (falling back to source compile with node-gyp)
```

and compiling yields above error, a known issue that has been fixed in later releases:
[https://github.com/grpc/grpc-node/issues/952#issuecomment-523626218](https://github.com/grpc/grpc-node/issues/952#issuecomment-523626218)

Please fix the dependency so npm can install the correct package version.

**Comment by jcramer** [comment]
Ok I have updated grpc-bchrpc-node pacakge to v0.7.0.  Can you see if this still an issue for you with 0.7.0?

**Comment by jcramer** [comment]
Closing this as I assume it was fixed in 0.7.0, we can reopen if needed/
